ethtools.js

```diff
@@ -236,7 +236,7 @@ EthTools.formatBalance = function(number, format, unit){
         var ticker = EthTools.ticker.findOne(unit, {fields: {price: 1}});
 
         // convert first to ether
-        number = web3.fromWei(number, 'ether');
+        number = web3.utils.fromWei(number.toString(), 'ether');
```

Comment:
Is that necessary? Shouldn't fromWei and toWei be accepting integers and not strings?

Reply:
It was throwing an error if the number was a BigNumber, since web3.js 1.0.0 mostly converted over to BN.

I was writing some conditional conversion logic, but decided toString kept it most simple 😊

Reply:
Wouldn't `toFixed`also work in both?

Reply:
Ah yes that seems smart, thanks. I’ll test and update